<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1234</width>
    <height>907</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>巡检机器人</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(8, 31, 78);</string>
   </property>
   <widget class="QWidget" name="video_widget" native="true">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>5</y>
      <width>326</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="video_label">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">border-width: 1px;
background-color: rgb(0, 0, 0);
border-style: solid;
border-color: rgb(0, 0, 0)
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="map_widget" native="true">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>616</width>
      <height>626</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="map_label">
       <property name="styleSheet">
        <string notr="true">border-width: 1px;
border-left-color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
border-style: solid;
border-color: rgb(0, 0, 0)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="robot_button_widget" native="true">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>774</y>
      <width>181</width>
      <height>86</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(32, 74, 135);</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="2" column="1">
      <widget class="QPushButton" name="right_down_Button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>down</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="right_right_Button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>right</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="right_left_Button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>left</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="right_up_Button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>up</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="ptz_button_widget" native="true">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>655</y>
      <width>176</width>
      <height>111</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(32, 74, 135);</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="1">
      <widget class="QPushButton" name="left_up_Button">
       <property name="text">
        <string>up</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="left_right_Button">
       <property name="text">
        <string>right</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="left_down_Button">
       <property name="text">
        <string>down</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="left_left_Button">
       <property name="text">
        <string>left</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="red_video_widget" native="true">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>315</y>
      <width>326</width>
      <height>316</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="red_video_label">
       <property name="styleSheet">
        <string notr="true">border-width: 1px;
background-color: rgb(0, 0, 0);
border-style: solid;
border-color: rgb(0, 0, 0)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="robot_status_widget" native="true">
    <property name="geometry">
     <rect>
      <x>975</x>
      <y>45</y>
      <width>200</width>
      <height>196</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(32, 74, 135);</string>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="horizontalSpacing">
      <number>40</number>
     </property>
     <property name="verticalSpacing">
      <number>20</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="link_status_label">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>链接状态:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="link_status_text_label">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>等待链接</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="battery_label">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>电池电量:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="battery_text_label">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>null</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="robot_speed_label">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>实时速度:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="robot_speed_text_label">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>null</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="charger_status_label">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>充电状态:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="charger_status_text_label">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>null</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="navigate_speed_label">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>导航速度:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="navigate_speed_text_label">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>低速</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="sensor_data_widget" native="true">
    <property name="geometry">
     <rect>
      <x>975</x>
      <y>280</y>
      <width>206</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(32, 74, 135);</string>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <property name="formAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="horizontalSpacing">
      <number>20</number>
     </property>
     <property name="verticalSpacing">
      <number>20</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="gas_one_">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>可燃气浓度1:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="gas_two_label">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>可燃气浓度2:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="gas_two_text_label">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>0%</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="security_status_label">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>安全状态:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="security_status_text_label">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(78, 154, 6);</string>
       </property>
       <property name="text">
        <string>安全</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="gas_one_text_label">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>0%</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="robot_control_widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>660</y>
      <width>951</width>
      <height>156</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="leftMargin">
      <number>30</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>30</number>
     </property>
     <property name="bottomMargin">
      <number>20</number>
     </property>
     <property name="horizontalSpacing">
      <number>40</number>
     </property>
     <property name="verticalSpacing">
      <number>10</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="control_display_label">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font: 57 16pt &quot;Ubuntu&quot;;</string>
       </property>
       <property name="text">
        <string>机器人控制</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QPushButton" name="PTZ_control_Button">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(233, 185, 110);</string>
       </property>
       <property name="text">
        <string>云台控制</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="navigate_stop_Button">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(239, 41, 41);</string>
       </property>
       <property name="text">
        <string>一键急停</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="navigate_back_Button">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(247, 231, 26);</string>
       </property>
       <property name="text">
        <string>一键返航</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="navigate_cruise_Button">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(10, 109, 248);</string>
       </property>
       <property name="text">
        <string>一键巡航</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="add_navigate_task_Button">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(115, 210, 22);</string>
       </property>
       <property name="text">
        <string>添加巡航任务点</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="delect_task_point_Button">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(92, 53, 102);</string>
       </property>
       <property name="text">
        <string>删除任务点</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="resume_navigate_task_Button">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(143, 89, 2);</string>
       </property>
       <property name="text">
        <string>恢复巡航</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QPushButton" name="Robot_control_Button">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(238, 238, 236);</string>
       </property>
       <property name="text">
        <string>机器人控制</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>415</y>
      <width>181</width>
      <height>226</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(238, 238, 236);
background-color: rgb(32, 74, 135);</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1234</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="map_menu">
    <property name="title">
     <string>地图</string>
    </property>
    <addaction name="new_map_act"/>
    <addaction name="separator"/>
    <addaction name="extend_map_act"/>
    <addaction name="separator"/>
    <addaction name="load_map_act"/>
    <addaction name="separator"/>
    <addaction name="delete_map_act"/>
   </widget>
   <widget class="QMenu" name="position_menu">
    <property name="title">
     <string>标记点</string>
    </property>
    <addaction name="add_position_act"/>
    <addaction name="separator"/>
    <addaction name="delete_position_act"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="navigate_menu">
    <property name="title">
     <string>导航</string>
    </property>
    <addaction name="separator"/>
    <addaction name="start_navigate_act"/>
    <addaction name="separator"/>
    <addaction name="pause_navigate_act"/>
    <addaction name="separator"/>
    <addaction name="resume_navigate_act"/>
    <addaction name="separator"/>
    <addaction name="cancel_navigate_act"/>
   </widget>
   <widget class="QMenu" name="speed_menu">
    <property name="title">
     <string>速度</string>
    </property>
    <addaction name="low_speed_act"/>
    <addaction name="separator"/>
    <addaction name="middle_speed_act"/>
    <addaction name="separator"/>
    <addaction name="high_speed_act"/>
   </widget>
   <addaction name="map_menu"/>
   <addaction name="position_menu"/>
   <addaction name="navigate_menu"/>
   <addaction name="speed_menu"/>
  </widget>
  <action name="new_map_act">
   <property name="text">
    <string>新建地图</string>
   </property>
  </action>
  <action name="extend_map_act">
   <property name="text">
    <string>扩展地图</string>
   </property>
  </action>
  <action name="load_map_act">
   <property name="text">
    <string>加载地图</string>
   </property>
  </action>
  <action name="delete_map_act">
   <property name="text">
    <string>删除地图</string>
   </property>
  </action>
  <action name="add_position_act">
   <property name="text">
    <string>添加标记点</string>
   </property>
  </action>
  <action name="delete_position_act">
   <property name="text">
    <string>删除标记点</string>
   </property>
  </action>
  <action name="start_navigate_act">
   <property name="text">
    <string>导航开始</string>
   </property>
  </action>
  <action name="pause_navigate_act">
   <property name="text">
    <string>导航暂停</string>
   </property>
  </action>
  <action name="resume_navigate_act">
   <property name="text">
    <string>恢复导航</string>
   </property>
  </action>
  <action name="cancel_navigate_act">
   <property name="text">
    <string>取消导航</string>
   </property>
  </action>
  <action name="low_speed_act">
   <property name="text">
    <string>低速</string>
   </property>
  </action>
  <action name="middle_speed_act">
   <property name="text">
    <string>中速</string>
   </property>
  </action>
  <action name="high_speed_act">
   <property name="text">
    <string>高速</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
